docs/website/docs/command-reference/completion.md

```diff
@@ -0,0 +1,81 @@
+---
+title: odo completion
+---
+
+## Description
+
+`odo completion` is used to generate shell completion code. The generated code provides interactive shell completion code for `odo`.
+
+There is support for the following terminal shell's:
+- [Bash](https://www.gnu.org/software/bash/)
+- [Zsh](https://ohmyz.sh/)
```

Comment:
Surely nitpicking, but I think we should link to the official zsh website instead. People can use other tools.

```suggestion
- [Zsh](https://zsh.sourceforge.io/)
```